ui.apps/src/main/content/jcr_root/apps/aemtraining/components/structure/StudentPage/customheaderlibs.html

```diff
@@ -21,6 +21,9 @@
 <sly data-sly-use.clientlib="core/wcm/components/commons/v1/templates/clientlib.html">
     <sly data-sly-call="${clientlib.css @ categories='Student-clientlib'}"/>
     <sly data-sly-call="${clientlib.css @ categories='ProductDetailLibs'}"/>
-    <sly data-sly-call="${clientlib.css @ categories='ClientReportLibs'}"/>
+
+
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
```

Comment:
Do we have any other way to include jquery, because in this case every time of page load a request call will go.